src/features/event/editor/editor.tsx

```diff
@@ -115,24 +157,24 @@ export default function EventEditor({ type, initialData }: EventEditorProps) {
   return (
     <div className="flex min-h-dvh flex-col space-y-4 pl-6 pr-6">
       <HeaderSpacer />
-      <div className="flex w-full items-center justify-between">
+
+      {/* Rate Limit Error */}
+      {errors.rate_limit && (
+        <Banner type="error" title="Woah! Slow down" className="mb-8 w-full">
+          {errors.rate_limit}
+        </Banner>
+      )}
+
+      <div className="-mb-1 flex w-full items-center justify-between">
         <div className="md:w-1/2">
-          <p
-            className={`text-error text-right text-xs ${errors.title ? "visible" : "invisible"}`}
-          >
-            {errors.title ? errors.title : "Error Placeholder"}
-          </p>
-          <input
+          <TextInputField
+            id={"event-name"}
             type="text"
+            label="Enter Event Name"
```

Comment:
```suggestion
            label="Event Name"
```
Seeing that the label will remain even after a value is entered, this label would make more sense.